lib/challenge.js

```diff
@@ -60,6 +86,20 @@ var some_pet;
 
 var Pet;
 // Answer Starts Here
+var Pet = function getPetInfo(name, species, breed, noise) = {
+  this.name = name;
+  this.species = species;
+  this.breed = breed;
+  this.noise = noise
+  this.makeNoise = makeNoise();
+}
+
+var Pet = New getPetInfo();
+
+Pet.prototype.makeNoise = function makeNoise() {
```

Comment:
Ah, ok, if you have a few minutes later, I would like to go over it just to
make sure I get it right.

On Mon, Oct 5, 2015 at 3:39 PM, Matt Brendzel <notifications@github.com>
wrote:

> In lib/challenge.js
> <https://github.com/ga-wdi-boston/js-objects-intro-assessment/pull/6#discussion_r41187305>
> :
>
> > @@ -60,6 +86,20 @@ var some_pet;
> >
> >  var Pet;
> >  // Answer Starts Here
> > +var Pet = function getPetInfo(name, species, breed, noise) = {
> > +  this.name = name;
> > +  this.species = species;
> > +  this.breed = breed;
> > +  this.noise = noise
> > +  this.makeNoise = makeNoise();
> > +}
> > +
> > +var Pet = New getPetInfo();
> > +
> > +Pet.prototype.makeNoise = function makeNoise() {
>
> This part is correct!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ga-wdi-boston/js-objects-intro-assessment/pull/6/files#r41187305>
> .
>



-- 
Regards,
Jeff Meyer

857-206-3572